packages/dropdowns/src/elements/Menu/Menu.tsx

```diff
@@ -22,18 +22,23 @@ import {
 import { MenuContext } from '../../utils/useMenuContext';
 
 interface IMenuProps extends HTMLAttributes<HTMLUListElement> {
-  /** See Popper [documentation](https://popper.js.org/docs/v2/modifiers/) for details */
+  /** Sets the [Popper](https://popper.js.org/docs/v2/modifiers/) modifiers */
   popperModifiers?: Modifiers;
+  /** Determines if events are enabled */
   eventsEnabled?: boolean;
+  /** Sets the z-index */
   zIndex?: number;
-  /**
-   * These placements differ from the default naming of Popper placements to
-   * accommodate RTL layouts
-   **/
+  /** Sets the [Popper](https://popper.js.org/docs/v2/modifiers/) placement. This property differs
+   * from the default Popper placements and accommodates RTL layouts.
+   */
   placement?: GARDEN_PLACEMENT;
+  /** Determines if the menu is animated */
```

Comment:
Perhaps go into a bit more detail. What about the menu is animated? (The appearance and dismissal.)

Reply:
Fixed in latest push.